dbld/make

```diff
@@ -4,3 +4,4 @@ set -e
 
 cd /build
 make "$@"
+make install
```

Comment:
hmm, why are we adding an explicit "install" after we already run what the user requested?

e.g. ./dbld/rules make-check  would execute this with "check", thus the make shell script would be executed with "check", and with the above change we would be installing as well. Is there a reason?

Thanks

Reply:
Thanks for the note and the example. 
Before the patch the "$*" arg (from rules->make) was used  to pass the "install" into the make script. But in this patch I wanted to use the "$*" for the defination of the image/platform (see my last comment).